barretenberg/cpp/src/barretenberg/benchmark/protogalaxy_bench/protogalaxy.bench.cpp

```diff
@@ -73,8 +73,10 @@ void fold_k(State& state) noexcept
         decider_pks.emplace_back(decider_pk);
         decider_vks.emplace_back(decider_vk);
     }
+    std::shared_ptr<typename ProtogalaxyProver::Transcript> transcript =
```

Comment:
Small C++ comment because I've seen these from Federico: in cases like this where the type is clear from context, the use of `auto` is preferred, AFAIK. That is, it's cleaner to do:

`auto transcript = std::make_shared<typename ProtogalaxyProver::Transcript>();`

In general, however, `auto` should only be used when the type is explicitly clear from context, or when absolutely needed (eg, [this](https://en.cppreference.com/w/cpp/language/structured_binding.html))

tagging @ledwards2225 just in case you are following sth else, not to override you.